xs2a-adapter-rest-server/src/main/java/de/adorsys/xs2a/adapter/controller/AspspService.java

```diff
@@ -19,12 +19,16 @@
 import de.adorsys.xs2a.adapter.model.AspspTO;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 
 import java.util.List;
 
 public interface AspspService {
-    @RequestMapping("/v1/aspsps")
+
```

Comment:
I think this code should be in `AspspController` and `AspspService` is removed. It's confusing in it's implements relation to controller compared to `AspspSearchService` and `AspspModifyService`.

Reply:
I do not agree. We need such an interface for Feign remote client. I will rename it to AspspApi.

Reply:
In this case why the controller you've added doesn't have an interface?

Reply:
We do not want to expose modifying operations. They will be used only with `dev` profile